__tests__/accountAbstraction.test.ts

```diff
@@ -0,0 +1,461 @@
+// tests of handling of accounts including account abstraction about hashes and signatures
+import {
+  AbstractionFunctions,
+  Account,
+  ArraySignatureType,
+  BigNumberish,
+  Call,
+  CallData,
+  Calldata,
+  Contract,
+  DeclareSignerDetails,
+  DeployAccountSignerDetails,
+  Provider,
+  RawCalldata,
+  Signature,
+  Signer,
+  TypedData,
+  cairo,
+  constants,
+  ec,
+  encode,
+  hash,
+  typedData,
+} from '../src';
+import {
+  compiledAccountAbstractionCasm,
+  compiledAccountAbstractionSierra,
+  compiledDummy1Casm,
+  compiledDummy1Sierra,
+  compiledDummy2Casm,
+  compiledDummy2Sierra,
+  compiledErc20Echo,
+  getTestAccount,
+  getTestProvider,
+} from './fixtures';
+import { initializeMatcher } from './schema';
+
+describe('Test account abstraction of Cairo 1 account', () => {
+  const provider = new Provider(getTestProvider());
+  const account0 = getTestAccount(provider);
+  const privateKeyAbstraction = '0x7aadb6605c9538199797920884694b5ce84fc68f92c832b0';
+  const starkKeyPubAbstraction = ec.starkCurve.getStarkKey(privateKeyAbstraction);
+  const fullPublicKey = encode.addHexPrefix(
+    encode.buf2hex(ec.starkCurve.getPublicKey(privateKeyAbstraction, false))
+  );
+  const ethAddress = '0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7';
```

Comment:
we have this const also in `utils.test.ts`, maybe worth now extracting the const and import it in both places?

Reply:
OK. will be done.